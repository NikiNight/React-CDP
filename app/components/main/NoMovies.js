import React from 'react';

const NoMovies = () => {
    return (
        <div className="no-movies">
            <p className="no-movies__title">No films found</p>
        </div>
    )
}

export default NoMovies;