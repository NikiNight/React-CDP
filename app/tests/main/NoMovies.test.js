import React from 'react';
import NoMovies from './../../components/main/NoMovies';
import renderer from 'react-test-renderer';

describe('Test NoMovies component', () => {

    test('NoMovies matches snapshot', () => {
        const tree = renderer.create(<NoMovies />)
          .toJSON();
        expect(tree).toMatchSnapshot();
    });

})