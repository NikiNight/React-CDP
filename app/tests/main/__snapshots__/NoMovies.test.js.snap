// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test NoMovies component NoMovies matches snapshot 1`] = `
<div
  className="no-movies"
>
  <p
    className="no-movies__title"
  >
    No films found
  </p>
</div>
`;
